extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -131,7 +131,11 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     if (this.plpContainer_.querySelector('.i-amphtml-amp-story-shopping-plp')) {
       return;
     }
-    const shoppingData = this.storeService_.get(StateProperty.SHOPPING_DATA);
+
+    const pageElement = this.element.parentElement;
+    const shoppingData = this.storeService_.get(StateProperty.SHOPPING_DATA)[
+      pageElement.id
+    ];
```

Comment:
This doesn't specifically happens in this PR, but how do you know the data is already loaded and available here?

Reply:
The template updates whenever shopping_data updates. It will always be up do date. 
This was part of the PDP PR which needs to be merged into this branch.

Reply:
The template updates whenever shopping_data updates. It will always be up do date.
This was part of the PDP PR which will be merged into this branch.